src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -180,14 +180,7 @@ public static LoggingConfiguration AppConfig
         /// Did the <see cref="Initialize"/> Succeeded? <c>true</c>= success, <c>false</c>= error, <c>null</c> = initialize not started yet.
         /// </summary>
         public bool? InitializeSucceeded { get; private set; }
-
-
-        /// <summary>
-        /// Gets or sets a value indicating whether the configuration files
-        /// should be watched for changes and reloaded automatically when changed.
-        /// </summary>
-        public bool AutoReload { get; set; }
```

Comment:
Can you tell me why this property is not needed anymore? it's now a breaking change!